app.js

```diff
@@ -49,6 +50,31 @@ setInterval(() => {
   });
 }, 60 * 60 * 1000);
 
+async function sendReminderEmail() {
+  const users = await db.users.findAll({
+    where: { updated_at: { [Op.lt]: moment().subtract(7, 'days').toDate() } },
+  });
+
+  users.map(async (user) => {
+    const mailToken = jwt.sign({
+      type: 'create_account',
+      email: user.email,
+    }, process.env.JWT_SECRET, { expiresIn: '7d' });
+    await mail.send(user.email, 'create_account', {
+      url: `${process.env.CREATE_ACCOUNT_URL}/create-account?token=${mailToken}`,
+    });
+    await db.users.update({
+      updated_at: new Date(),
+    }, { where: { email: user.email } });
+  });
+}
+setInterval(() => {
+  logger.debug('running email reminder');
+  sendReminderEmail().catch((error) => {
+    logger.error(error, 'error sending email reminder');
+  });
+}, 60 * 60 * 1000);
```

Comment:
Offset this from the cleanup db timer, bad luck having them both run at the same time